# *****************************************************************
#
# HCL Confidential
#
# OCO Source Materials
#
# Copyright HCL Technologies Limited 2008, 2019
#
# The source code for this program is not published or otherwise
# divested of its trade secrets, irrespective of what has been
# deposited with the U.S. Copyright Office.
#
# *****************************************************************

# NLS_ENCODING=UNICODE
# NLS_MESSAGEFORMAT_VAR
## G11N SA UI


# COMPONENTPREFIX CLFRQ
str_component_id=CLFRQ

error.service.dboard.rollback=CLFRQ0114E: Ocorreu um erro durante a recupera\u00e7\u00e3o da transa\u00e7\u00e3o do banco de dados.

# The following properties contain non-readable non-translatable parameters (such as '7b1a60f1xc9cax4cc4x8b0bx51af2ddef2cd')
#error.UserPrefServices.getUserPrefs=CLFRQ0115E: An error occurred while retrieving user preferences information with this information: {0}
#error.UserPrefServices.getByMail=CLFRQ0120E: An error occurred while retrieving user preferences by email with this information: {0}, {1}
#error.UserPrefServices.getByID=CLFRQ0118E: An error occurred while retrieving user preferences by identifier with this information: {0}, {1}
#error.UserPrefServices.getByPosition=CLFRQ0116E: An error occurred while retrieving user preferences by position with this information: {0},{1},{2},{3}
#error.UserPrefServices.findByPageIDColNum=CLFRQ0119E: An error occurred while retrieving user preferences by page and column with this information: {0},{1},{2}
#error.UserPrefServices.findByPageIDRowNum=CLFRQ0117E: An error occurred while retrieving user preferences by page and row with this information: {0},{1},{2}
#error.UserPrefServices.createByMail=CLFRQ0121E: An error occurred while creating user preferences by email with this information: {0},{1},{2},{3},{4},{5},{6}
#error.UserPrefServices.createByID=CLFRQ0135E: An error occurred while creating user preferences by user identifier with this information: {0},{1},{2},{3},{4},{5},{6}
error.UIServices.changePosition=CLFRQ0138E: Ocorreu um erro durante a altera\u00e7\u00e3o da posi\u00e7\u00e3o para um widget com estas informa\u00e7\u00f5es: {0},{1},{2},{3}
error.UIServices.changeWidgetSetting=CLFRQ0132E: Ocorreu um erro durante a altera\u00e7\u00e3o de configura\u00e7\u00f5es para um widget com estas informa\u00e7\u00f5es: {0},{1}
#error.UserPrefServices.changeWidgetDisplay=CLFRQ0136E: An error occurred while changing display for a widget with this information: {0}, {1}, {2}
#error.UserPrefServices.deleteByID=CLFRQ0129E: An error occurred while deleting widget preferences by user identifier with this information: {0}, {1}
#error.UserPrefServices.deleteByMail=CLFRQ0130E: An error occurred while deleting widget preferences by user email with this information: {0},{1}
#error.UserPrefServices.getUserWidgetPref=CLFRQ0140E: An error occurred while retrieving widget preferences with this identifier: {0}
error.UIServices.deleteWidgetInstance=CLFRQ0399E: Ocorreu um erro durante a exclus\u00e3o da inst\u00e2ncia de widget com esta chave prim\u00e1ria: {0}


#error.UIServices.createUserPanel=CLFRQ0294E: An  error occurred while retrieving the information related to the user interface for this user: {0}
error.UIServices.createUserInstanceWidgets=CLFRQ0292E: ocorreu um erro ao criar um widget para o usu\u00e1rio: {0}. Aqui est\u00e3o as informa\u00e7\u00f5es do widget que o sistema n\u00e3o pode armazenar. ID do Widget: {1}, Ordem: {2}, Cont\u00eainer: {3}, IsToggled: {4}, PagePanel id {5}, Configura\u00e7\u00f5es do Widget: {6}
error.UIServices.getUserWidgetInst=CLFRQ0291E:  Ocorreu um erro durante a recupera\u00e7\u00e3o de prefer\u00eancias do widget com este identificador: {0}
#error.UIServices.getUserWidgetsInstances=CLFRQ0293E:  An error occurred while retrieving the widget instances for the user: {0} related to this panel page {1}
error.UIServices.getUserWidgetInstanceByWidgetId=CLFRQ0295E:  Ocorreu um erro durante a recupera\u00e7\u00e3o de inst\u00e2ncias do widget para o usu\u00e1rio: {0} relacionado a este painel de p\u00e1gina {1} e este ID de widget {2}


error.UserServices.getUserByID=CLFRQ0133E: Ocorreu um erro durante a recupera\u00e7\u00e3o de informa\u00e7\u00f5es do usu\u00e1rio com este identificador: {0}
error.UserServices.getUserByEmail=CLFRQ0131E: Ocorreu um erro durante a recupera\u00e7\u00e3o de informa\u00e7\u00f5es sobre o usu\u00e1rio com este e-mail: {0}
error.UserServices.createUser=CLFRQ0128E: Ocorreu um erro durante a cria\u00e7\u00e3o de um usu\u00e1rio com estas informa\u00e7\u00f5es: {0}
error.UserServices.createUserUI=CLFRQ0384E: Ocorreu um erro durante a cria\u00e7\u00e3o de um usu\u00e1rio com este ID interno: {0}
error.UserServices.updateWelcomeNote=CLFRQ0384A: Ocorreu um erro durante a atualiza\u00e7\u00e3o da nota de boas-vindas para este ID interno: {0}

error.UserServices.getNumberUsers=CLFRQ0134E: Ocorreu um erro ao recuperar o n\u00famero de usu\u00e1rios registrados no banco de dados.
warning.UserServices.updateLastVisitForUser=CLFRQ0139W: Ocorreu um erro durante a atualiza\u00e7\u00e3o da \u00faltima visita para um usu\u00e1rio com estas informa\u00e7\u00f5es: {0},{1}
warning.UserServices.updateLanguageForUser=CLFRQ0398W: Ocorreu um erro durante a atualiza\u00e7\u00e3o do idioma para um usu\u00e1rio com estas informa\u00e7\u00f5es: {0},{1}
warning.UserServices.updateWelcome=CLFRQ0137W: Ocorreu um erro durante a atualiza\u00e7\u00e3o do modo de boas-vindas para um usu\u00e1rio com estas informa\u00e7\u00f5es:{0},{1}
error.UIServices.get.tabInstance=CLFRQ0348E: Ocorreu um erro durante a obten\u00e7\u00e3o da inst\u00e2ncia da guia com o ID {0}. Verifique a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.UIServices.updateNumberColumn.dataAccess=CLFRQ0349E: ocorreu um erro durante a atualiza\u00e7\u00e3o do n\u00famero de colunas para a inst\u00e2ncia da guia {0}. Verifique a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.UIServices.getAllTabInstancesForPerson.rollback=CLFRQ0347E: N\u00e3o foi poss\u00edvel recuperar uma transa\u00e7\u00e3o ao criar inst\u00e2ncias da guia para o ID do usu\u00e1rio {0}. Verifique a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.UIServices.getAllTabInstancesForPerson.access=CLFRQ0345E: Ocorreu um erro durante a cria\u00e7\u00e3o de uma inst\u00e2ncia da guia para o ID do usu\u00e1rio {0}. Verifique a exce\u00e7\u00e3o aninhada para obter detalhes.
error.UIServices.getAllTabInstancesForPerson.retrieve=CLFRQ0397E: Ocorreu um erro ao buscar a lista de todas as inst\u00e2ncias da guia para o ID do usu\u00e1rio {0}. Verifique a exce\u00e7\u00e3o aninhada para obter detalhes.

error.admin.testmode.disable=CLFRQ0189E: O modo de teste de administra\u00e7\u00e3o n\u00e3o p\u00f4de ser desativado para o usu\u00e1rio com o e-mail {0}.
error.admin.testmode.enable=CLFRQ0188E: O modo de teste de administra\u00e7\u00e3o n\u00e3o p\u00f4de ser ativado para o usu\u00e1rio com o e-mail {0}.
error.admin.testmode.check=CLFRQ0190E: O status atual do modo de teste de administra\u00e7\u00e3o n\u00e3o p\u00f4de ser recuperado para o usu\u00e1rio com o e-mail {0}.

error.mbean.person.service.dsservice=CLFRQ0221E: A refer\u00eancia para os servi\u00e7os de diret\u00f3rio n\u00e3o p\u00f4de ser recuperada.
error.mbean.person.service.daofactory=CLFRQ0223E: Uma refer\u00eancia para o factory do objeto de acesso a dados n\u00e3o p\u00f4de ser recuperada.
error.mbean.person.service.dataaccess=CLFRQ0220E: Ocorreu um erro durante o acesso ao banco de dados: {0}
error.mbean.person.service.dserror=CLFRQ0218E: Ocorreu um erro durante o acesso aos servi\u00e7os de diret\u00f3rio: {0}

# {0} contains a file name
error.mbean.person.service.filenotfound=CLFRQ0219E: O arquivo [{0}] n\u00e3o existe.

# {0} contains a file name, {1} contains an error message from an external service
error.mbean.person.service.fileerror=CLFRQ0222E: Ocorreu um erro ao tentar ler o arquivo [{0}]: {1}

# {0} contains a user id or email
error.mbean.person.service.nouser=CLFRQ0227E: O usu\u00e1rio com o e-mail [{0}] n\u00e3o est\u00e1 registrado no banco de dados da P\u00e1gina Inicial.
info.mbean.person.service.ok=CLFRQ0224I: A atualiza\u00e7\u00e3o foi conclu\u00edda com sucesso.
# {0} and {1} are integers
warn.mbean.person.service.notallfound=CLFRQ0226W: Nem todos os usu\u00e1rios foram localizados no banco de dados da P\u00e1gina Inicial ou no servi\u00e7o de diret\u00f3rio do HCL Connections. {0} de {1} usu\u00e1rios foram atualizados. Verifique os logs do servidor para obter detalhes.
warn.mbean.person.service.nodsobject=CLFRQ0225W: o usu\u00e1rio com o e-mail [{0}] n\u00e3o foi encontrado no servi\u00e7o de diret\u00f3rio. Este usu\u00e1rio pode n\u00e3o existir mais.
warn.mbean.person.service.nodsobject.ln=CLFRQ0335W: O usu\u00e1rio com o nome de login [{0}] n\u00e3o foi encontrado no servi\u00e7o de diret\u00f3rio. Este usu\u00e1rio pode n\u00e3o existir mais.
warn.mbean.person.service.nodsobject.id=CLFRQ0332W: O usu\u00e1rio com o ID [{0}] n\u00e3o foi encontrado no servi\u00e7o de diret\u00f3rio. Este usu\u00e1rio pode n\u00e3o existir mais.
# {0} is a login name
error.mbean.person.service.loginname.exception=CLFRQ0411E: Ocorreu uma exce\u00e7\u00e3o ao atualizar o usu\u00e1rio com o nome de login {0}. Verifique a exce\u00e7\u00e3o aninhada para obter mais detalhes.
# {0} is an email
error.mbean.person.service.email.exception=CLFRQ0406E: Ocorreu uma exce\u00e7\u00e3o ao atualizar o usu\u00e1rio com o e-mail {0}. Verifique a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.mbean.person.service.email.notfound.hp=CLFRQ0407E: O usu\u00e1rio com o endere\u00e7o de e-mail {0} n\u00e3o p\u00f4de ser encontrado no banco de dados da P\u00e1gina Inicial.
error.mbean.person.service.loginname.notfound.hp=CLFRQ0409E: O usu\u00e1rio com o nome de login {0} n\u00e3o p\u00f4de ser encontrado no banco de dados da P\u00e1gina Inicial.

# HomepageMBeanFacade
info.invocation.start=CLFRQ0615I: Chamar servi\u00e7o [{0}], m\u00e9todo [{0}]
info.invocation.complete=CLFRQ0614I: Conclu\u00edda a chamada do servi\u00e7o [{0}], m\u00e9todo [{1}]"
error.service.not.exist=CLFRQ0617E: N\u00e3o \u00e9 poss\u00edvel localizar o servi\u00e7o [{0}] no registro
error.service.no.access=CLFRQ0618E: N\u00e3o \u00e9 poss\u00edvel acessar o servi\u00e7o [{0}] no registro
error.service.cant.construct=CLFRQ0616E: N\u00e3o \u00e9 poss\u00edvel construir a inst\u00e2ncia do servi\u00e7o [{0}] no registro
error.method.not.found=CLFRQ0613E: N\u00e3o \u00e9 poss\u00edvel chamar o m\u00e9todo [{1}] no servi\u00e7o [{0}]

# Metrics stuff

# Translator: MBean should not be translated
error.mbean.registrar=CLFRQ0243E: Ocorreu um erro ao registrar o MBean da P\u00e1gina Inicial. Os recursos de administra\u00e7\u00e3o podem n\u00e3o estar dispon\u00edveis.
error.mbean.unregistering=CLFRQ0269E: Ocorreu um erro ao remover o registro do MBean da P\u00e1gina Inicial.
error.mbean.locate=CLFRQ0246E: O servidor MBean n\u00e3o p\u00f4de ser localizado.
error.metrics.statistics.file.creating=CLFRQ0253E: Ocorreu um erro durante a cria\u00e7\u00e3o do arquivo de estat\u00edsticas da P\u00e1gina Inicial.
error.metrics.statistics.file.writing=CLFRQ0242E: Ocorreu um erro durante a grava\u00e7\u00e3o do arquivo de estat\u00edsticas da P\u00e1gina Inicial.
error.metrics.statistics.file.closing=CLFRQ0235E: Ocorreu um erro durante o fechamento do arquivo de estat\u00edsticas da P\u00e1gina Inicial.
error.metrics.statistics.file.parsing=CLFRQ0252E: Ocorreu um erro durante a an\u00e1lise do arquivo de estat\u00edsticas da P\u00e1gina Inicial.
error.metrics.statistics.getNumberVisitPast=CLFRQ0315E: Ocorreu um erro durante a recupera\u00e7\u00e3o do n\u00famero de visitas exclusivas iniciando a partir desta data: {0}

info.metrics.property.value=CLFRQ0257I: Nome da propriedade: {0}, valor da propriedade: {1}

info.metrics.setting.timeout=CLFRQ0241I: Configurando o tempo limite para {0} minutos.
info.metrics.setting.separator=CLFRQ0250I: O servi\u00e7o de m\u00e9tricas est\u00e1 usando este separador: [{0}]
info.metrics.ok.cache.timeout=CLFRQ0240I: O tempo limite foi corretamente alterado para este valor: [{0}]
error.metrics.cache.timeout=CLFRQ0247E: O valor do tempo limite de cache n\u00e3o \u00e9 v\u00e1lido. Ele \u00e9 menor do que 0 ou superior a 1440 minutos.
error.metrics.bad.sample.count=CLFRQ0261E: O valor da contagem de amostra n\u00e3o \u00e9 v\u00e1lido.
error.metrics.bad.separator=CLFRQ0254E: O comprimento do separador est\u00e1 errado. Ele \u00e9 maior do que um caractere.
error.metrics.field.key=CLFRQ0266E: O tipo de dados da chave do campo est\u00e1 errado.
error.metrics.field.key.notfound=CLFRQ0245E: O valor da chave do campo est\u00e1 errado.
error.metrics.file=CLFRQ0236E: O arquivo usado no servi\u00e7o de m\u00e9tricas n\u00e3o \u00e9 v\u00e1lido. O nome do arquivo \u00e9 [{0}].
info.metrics.dir=CLFRQ0264I: Testando se o arquivo ou diret\u00f3rio [{0}] existe.
info.metrics.dir.ok=CLFRQ0256I: O arquivo ou diret\u00f3rio [{0}] existe.
info.metrics.file.ok=CLFRQ0268I: O caminho [{0}] \u00e9 um arquivo, n\u00e3o um diret\u00f3rio.
info.metrics.file.ok.read=CLFRQ0244I: O arquivo [{0}] pode ser lido.
error.metrics.file.read=CLFRQ0239E: O arquivo [{0}] n\u00e3o pode ser lido.
error.metrics.notfile=CLFRQ0260E: Um diret\u00f3rio foi especificado, mas n\u00e3o um arquivo.
error.metrics.file.notexisting=CLFRQ0255E: O arquivo ou diret\u00f3rio {0} n\u00e3o existe.
info.metrics.file.success=CLFRQ0262I: Arquivo [{0}] criado com sucesso.
error.metrics.file.create=CLFRQ0259E: A cria\u00e7\u00e3o do arquivo [{0}] falhou.
error.metrics.file.already.exists=CLFRQ0248E: O arquivo [{0}] j\u00e1 existe.
info.metrics.hash.seeded=CLFRQ0237I: A tabela de dispers\u00e3o possui valores iniciais.

error.metrics.total.users=CLFRQ0267E: O valor do n\u00famero total de usu\u00e1rios n\u00e3o p\u00f4de ser recuperado.
error.metrics.total.widgets=CLFRQ0251E: O valor do n\u00famero total de widgets n\u00e3o p\u00f4de ser recuperado.
error.metrics.enabled.widgets=CLFRQ0258E: O valor do n\u00famero total de widgets ativados n\u00e3o p\u00f4de ser recuperado.
error.metrics.open.widgets=CLFRQ0263E: O valor do n\u00famero total de widgets abertos n\u00e3o p\u00f4de ser recuperado.
error.metrics.widgets.usage=CLFRQ0265E: O valor do n\u00famero de widgets usados n\u00e3o p\u00f4de ser recuperado.
error.metrics.visits.hour=CLFRQ0238E: n\u00e3o foi poss\u00edvel recuperar o valor do n\u00famero de visitas na \u00faltima hora.
error.metrics.visits.day=CLFRQ0249E: O valor do n\u00famero de visitas nas \u00faltimas 24 horas n\u00e3o p\u00f4de ser recuperado.

error.NotificationService.addNotification=CLFRQ0314E: Ocorreu um erro durante a cria\u00e7\u00e3o de uma notifica\u00e7\u00e3o com estas informa\u00e7\u00f5es: {0}

error.widgetInfo.retrieve.HttpError=CLFRQ0322E: Ocorreu um erro durante a recupera\u00e7\u00e3o do descritor do widget localizado em {0}. O c\u00f3digo de erro retornado foi {1}
error.widgetInfo.retrieve.IOError=CLFRQ0319E: ocorreu um erro ao ler o descritor do widget retornado localizado em {0}. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.widgetInfo.retrieve.IOError.file=CLFRQ0320E: Ocorreu um erro ao ler o arquivo descritor do widget localizado em {0}. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.widgetInfo.retrieve.http.HttpError=CLFRQ0321E: Ocorreu um erro HTTP ao recuperar o descritor do widget localizado em {0}. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes.
error.widgetInfo.retrieve.http.IOError=CLFRQ0318E: Ocorreu um erro de E/S ao recuperar o descritor do widget localizado em {0}. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes.
warn.cache.retrieve.batch=CLFRQ0396W: Ocorreu um erro durante o armazenamento do descritor para o widget {0} em um processo em lote. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes. Processo em lote continuando.

# Translator: {0} contains an email address, {1} contains a user ID
error.email.duplicate.exid=CLFRQ0336E: O e-mail [{0}] j\u00e1 existe no banco de dados. O usu\u00e1rio identificado com o ID externo [{0}] n\u00e3o pode ser inclu\u00eddo no banco de dados.
error.email.duplicate.loginname=CLFRQ0334E: O e-mail [{0}] j\u00e1 existe no banco de dados. O usu\u00e1rio identificado com o nome de login [{0}] n\u00e3o pode ser inclu\u00eddo no banco de dados.
error.empty.dsobject=CLFRQ0333E: um DSObject vazio foi enviado como um par\u00e2metro.

error.SecuritySupportServices.DAOException=CLFRQ0391E: ocorreu um erro na camada de servi\u00e7o de seguran\u00e7a ao tentar executar uma opera\u00e7\u00e3o relacionada ao banco de dados. Consulte a exce\u00e7\u00e3o aninhada para obter mais detalhes.

info.homepagepersonservice.result=CLFRQ0410I: A opera\u00e7\u00e3o foi conclu\u00edda com \u00eaxito. O n\u00famero de registros atualizados \u00e9 {0}.
